wearApp/src/main/java/dev/johnoreilly/confetti/wear/settings/SettingsListView.kt

```diff
@@ -36,7 +40,25 @@ fun SettingsListView(
         }
 
         item {
-            StandardChip(label = "Change Conference", onClick = conferenceCleared)
+            StandardChip(
+                label = "Change Conference",
```

Comment:
nit: add texts from this file to strings resources file

Reply:
Yep, I think that's a wider problem.